TestLoopX/src/testloop.cxx

```diff
@@ -1,4 +1,5 @@
 #include <cctk.h>
+#include "CarpetX/CarpetX/src/driver.hxx"
```

Comment:
I agree with your suggestions, thanks for pointing them out. In order to use the `USES INCLUDE` mechanism for `driver.hxx`, this would require adding a `INCLUDES HEADER: driver.hxx IN driver.hxx` to the `interface.ccl` for CarpetX.